.github/workflows/e2e-test-setup.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright The Notary Project Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: e2e-test-setup
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  e2e-test-setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # E2E test cases on Notation: Setup
+      - name: Setup Notation
+        uses: notaryproject/notation-action/setup@main
+    
+      - name: Setup Notation using unsupported version
+        id: unsupported-version
+        continue-on-error: true
+        uses: notaryproject/notation-action/setup@main
+        with:
+            version: unsupported
+      - name: 'Should Fail: Setup Notation using unsupported version'
+        if: steps.unsupported-version.outcome != 'failure'
+        run: |
+            echo "Setup Notation using unsupported version should fail, but succeeded."
+            exit 1
+
+      - name: Setup Notation using URL
+        uses: notaryproject/notation-action/setup@main
+        with:
+            url: https://github.com/notaryproject/notation/releases/download/v1.0.0-rc.6/notation_1.0.0-rc.6_linux_amd64.tar.gz
```

Comment:
We can use `1.0.0` now

Reply:
This will be done in a follow up PR. Because there are other changes related to `version` in the code.

Reply:
If it will be done in the PR, could you create an issue for tracking?

Reply:
> If it will be done in the PR, could you create an issue for tracking?

Sure. @FeynmanZhou has created an issue related to it. Referencing that issue here: https://github.com/notaryproject/notation-action/issues/36